vale/styles/jargonLint/Technology.yml

```diff
@@ -6,256 +6,361 @@ ignorecase: true
 level: warning
 message: Use '%s' instead of '%s'
 swap:
+  # Disable yamlint rule about line length
+  # yamllint disable-line rule:line-length
+  #
+  # Sources
+  # - https://github.com/sapegin/textlint-rule-terminology/blob/master/terms.jsonc
+  # - https://github.com/paradite/frontend-encyclopedia/blob/main/scripts/data.json
+  # - https://github.com/huacnlee/autocorrect/blob/main/.autocorrectrc.template
+  # - many others
+  #
+  # Restore yamlint rule about length
+  # yamllint enable rule:line-length
+  #
+  # Ad hoc: Ad hoc # risk of false positives
```

Comment:
I didn't want to add this, again...

Reply:
I also don't like it, we should maintain sources in an other file.

Reply:
I will close this PR